aip/general/0121.md

```diff
@@ -94,6 +94,37 @@ the standard methods. Custom methods offer the same design freedom as
 traditional RPC APIs, which can be used to implement common programming
 patterns, such as database transactions, import and export, or data analysis.
 
+### Strong Consistency
+
+For methods that operate on the [managment plane][], the completion of those
+operations (either successful or with an error, LRO or synchronous) **must**
+mean that the state of the resource's existence and all user-settable values
+have reached a steady-state.
+
+[output only][] values **should** also have reached a steady-state, except
```

Comment:
What does this mean?

Reply:
basically that output-only values should be stable if they aren't related to resource state. E.g. they're not jittering around in values. This is for the same reason the operation itself should be consistent: the user should expect resource creation marks some event where the resource is more or less stable, since declarative clients have trouble reconciling transient state, and users will be surprised if the resource is constantly changing.

Of course being related to resource state is a little vague, since the only definition of state we have is AIP-216.

Reply:
Like server generated IDs? create_time ?

Reply:
those are good examples of things that are steady state. AIP-216's state field is an example of what shouldn't be.

Would adding examples help? Or do you think this statement is too ambiguous?